internal/runners/languages/languages.go

```diff
@@ -51,10 +68,42 @@ func (l *Languages) Run() error {
 		return locale.WrapError(err, "err_fetching_languages", "Cannot obtain languages")
 	}
 
+	// Fetch resolved artifacts list for showing full version numbers.
+	rt, err := runtime.NewFromProject(l.project, target.TriggerLanguage, l.analytics, l.svcModel, l.out, l.auth, l.cfg)
+	if err != nil {
+		return locale.WrapError(err, "err_languages_runtime", "Could not initialize runtime")
```

Comment:
I know we're supposed to be shying away from `locale.*Error` functions in favor of user-facing errors, but this command doesn't yet have user-facing errors and implementing them here is too much for this ticket. Please let me know if there's a better way of handling this.